x-pack/plugins/security_solution/server/endpoint/services/actions/utils.test.ts

```diff
@@ -147,6 +148,42 @@ describe('When using Actions service utilities', () => {
         completedAt: COMPLETED_AT,
         errors: undefined,
         wasSuccessful: true,
+        outputs: {},
+      });
+    });
+
+    it('should return action outputs (if any) per agent id', () => {
+      const runningProcesses = endpointActionGenerator.randomResponseActionRunningProcesses(3);
+      const endpointResponse = endpointActionGenerator.generateActivityLogActionResponse({
+        item: {
+          data: {
+            '@timestamp': COMPLETED_AT,
+            agent: { id: '123' },
+            EndpointActions: {
+              completed_at: COMPLETED_AT,
+              output: {
+                type: 'json',
+                content: {
+                  entries: runningProcesses,
+                },
+              },
+            },
+          },
+        },
+      });
+      expect(getActionCompletionInfo(['123'], [endpointResponse])).toEqual({
+        isCompleted: true,
+        completedAt: COMPLETED_AT,
+        errors: undefined,
+        wasSuccessful: true,
+        outputs: {
+          '123': {
+            type: 'json',
+            content: {
+              entries: runningProcesses,
+            },
+          },
+        },
```

Comment:
This test will be obsolete once you revert the type change for `LogsEndpointActionResponse` type